release/release_workflow.wdl

```diff
@@ -18,7 +18,7 @@ task do_major_release {
      set -x 
 
      # Clone repo and checkout develop
-     git clone https://github.com/~{organization}/cromwell.git -b develop cromwell
+     git clone git@github.com:~{organization}/cromwell.git -b develop cromwell
```

Comment:
FYI @cjllanwarne just had changed it after the last release [here](https://github.com/broadinstitute/cromwell/pull/4198) from ssh to https. I don't know what the right way to do it is but it would be nice to figure out a reliable way to make it work for everyone.

Reply:
We're not sure why we went from ssh -> https, but it failed because the script paused waiting for stdin for the user to type their github username & password

It worked fine when we changed back to ssh

Reply:
I've always preferred SSH as well. I don't want to re-type my credentials every time I do anything - and for 2FA users, you can't use your password at the prompt, you have to use a access token.

Reply:
I changed this because the https version works out of the box for me whereas ssh fails... Sounds like you've had the opposite experience.

Perhaps I just set up the "auto authenticate with https" and everyone else has set up the "auto authenticate with ssh" 🤷‍♂️ 

FWIW the https version is what is copied to the clipboard from github when you want to clone a repo. But if I'm the minority here, we should go with ssh

Reply:
> FWIW the https version is what is copied to the clipboard from github when you want to clone a repo. 

FYI: 
<img width="438" alt="screen shot 2018-10-18 at 2 58 52 pm" src="https://user-images.githubusercontent.com/791985/47177444-6db9bc00-d2e6-11e8-9753-5a4e89e513b7.png">

Switching between HTTPS and SSH is a toggle that gets saved in your account.

Reply:
Ha, fair enough then!